addons/html_builder/static/src/core/building_blocks/builder_many2one.js

```diff
@@ -50,16 +51,22 @@ export class BuilderMany2One extends Component {
             const selectedString = getValue(el);
             const selected = selectedString && JSON.parse(selectedString);
             if (selected && !("display_name" in selected && "name" in selected)) {
-                Object.assign(
-                    selected,
-                    (
+                let value;
+                if (!selected.id) {
+                    value = {
+                        display_name: "Remote",
+                        name: "Remote",
+                    };
+                } else {
```

Comment:
You shouldn't hard code "Remote" in the generic component.
You should probably pass this data as a props. Why not use props.null_text  ?